### PR TITLE
chore: upgrade Django@2

### PR DESCRIPTION
**Pull request by obdulia-losantos**
- Removed Django 1 compatibility in `aether.common` module
- The `django-cas-ng` file is included in the dependencies folder because it is not in the pypi repository yet.
- The dependency `rest_framework_extensions` in Kernel was removed (it's not compatible yet) because was useless.

### CODE REVIEW

File: aether-common-module/README.md [hunk 1]
```diff
@@ -23,18 +23,6 @@ Includes the methods that allow:
...
-  from django.conf.urls import url
-  from aether.common.auth.views import obtain_auth_token
-
-
-  urlpatterns = [
-      url(r'^get-token$', obtain_auth_token, name='token'),
-  ]
-  ```
-
 - Django 2.x
```
Comment (admbtlr) [inline aether-common-module/README.md]: We should probably remove this `Django 2.x` comment too

File: aether-common-module/README.md [hunk 2]
```diff
@@ -67,18 +43,6 @@ Includes the methods that allow:
...
-  from django.conf.urls import url
-  from aether.common.health.views import health
-
-
-  urlpatterns = [
-      url(r'^health$', health, name='health'),
-  ]
-  ```
-
 - Django 2.x
```
Comment (admbtlr) [inline aether-common-module/README.md]: Likewise

File: aether-common-module/README.md [hunk 3]
```diff
@@ -111,18 +63,6 @@ Includes the methods that allow:
...
-  from django.conf.urls import url
-  from aether.common.kernel.views import check_kernel
-
-
-  urlpatterns = [
-      url(r'^check-kernel$', check_kernel, name='check-kernel'),
-  ]
-  ```
-
 - Django 2.x
```
Comment (admbtlr) [inline aether-common-module/README.md]: ...

File: aether-common-module/aether/common/conf/urls.py [hunk 4]
```diff
@@ -71,32 +41,27 @@ def generate_urlpatterns(token=False, kernel=False):  # pragma: no cover
...
-        admin_urls,
+        path('admin/', admin.site.urls),
 
         # `accounts` management
-        url_pattern(r'^accounts/', include(auth_urls, namespace='rest_framework')),
+        path('accounts/', include(auth_urls, namespace='rest_framework')),
 
         # media files (protected)
-        url_pattern(r'^media/(?P<path>.*)$', login_required(media_serve), name='media'),
+        path('media/<path:path>', login_required(media_serve), name='media'),
```
Comment (admbtlr) [inline aether-common-module/aether/common/conf/urls.py:61]: This is so much nicer!

**Review by admbtlr — CHANGES_REQUESTED** [review]
Just remove these comments, then I think we should do this. You happy to go to Django 2 @shawnsarwar ?

**Comment by obdulia-losantos** [comment]
@adamvert cleaned!

**Review by admbtlr — APPROVED** [review]

**Comment by obdulia-losantos** [comment]
@shawnsarwar whenever you want! :wink:

**Pull request merged**